config/zones.json

```diff
@@ -1524,12 +1524,49 @@
     ]
   },
   "US-NY": {
+    "capacity": {
+      "coal": 1113,
+      "gas": 11686,
+      "hydro": 3538,
+      "hydro storage": 1297,
+      "nuclear": 5292,
+      "oil": 8260,
+      "solar": 250,
+      "wind": 1010,
+      "biomass": 50
+    },
     "parsers": {
       "production": "US_NY.fetch_production"
     },
     "flag_file_name": "us.png",
     "contributors": [
       "https://github.com/systemcatch"
+      "https://github.com/hanshyde"
+    ]
+  },
+  "US-PJM": {
+    "capacity": {
+      "coal": 65205,
+      "gas": 63025,
+      "hydro": 8675,
+      "nuclear": 33164,
+      "oil": 9106,
+      "solar": 392,
+      "wind": 1114,
+      "biomass": 716,
+      "biogas": 904,
+      "other renewable": 42,
+      "demand response": 9112,
+      "energy efficiency": 1790,
+      "external": 196,
+      "unknown": 18,
```

Comment:
Unfortunately we only handle a given set of keys (so you have to remove/merge biogas, other renewable, demand response, energy efficiency and external)

Reply:
This comment hasn't been addressed yet.